src/main/java/seedu/address/logic/parser/tags/AddTagCommandParser.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.parser.tags;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.tags.AddTagCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new AddTagCommand object.
+ */
+public class AddTagCommandParser implements Parser<AddTagCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of an AddTagCommand.
```

Comment:
i think this sentence should end with "and returns an AddTagCommand object for execution."